apps/bsatool/CMakeLists.txt

```diff
@@ -18,3 +18,7 @@ if (BUILD_WITH_CODE_COVERAGE)
   add_definitions (--coverage)
   target_link_libraries(bsatool gcov)
 endif()
+
+if(MSVC)
+  target_link_libraries(bsatool imm32.lib winmm.lib version.lib)
```

Comment:
Is there a reason you are adding these, according to you, QT-related libraries to the non QT projects? We only use QT for the launcher, wizard and opencs.

Might be worth looking into why QT isn't adding those libraries on its own. Our other Windows developers don't seem to have a problem with these. What version of Qt are you using? dynamic or static libraries? Ideally the Qt cmake script should be handling all the needed dependencies.